pkgs/by-name/sh/shadps4/package.nix

```diff
@@ -0,0 +1,79 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  xorg,
+  qt6,
+  vulkan-headers,
+  vulkan-loader,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "shadps4";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "shadps4-emu";
+    repo = "shadPS4";
+    rev = "refs/tags/v.${finalAttrs.version}";
+    hash = "sha256-HNO/L2cMZVHTHczjKgUFVDDdWBnj7SKctGi+P7jATpQ=";
+    fetchSubmodules = true;
+  };
+
+  patches = [ ./bloodborne.patch ]; # https://github.com/shadps4-emu/shadPS4/issues/758
+
+  buildInputs = [
+    xorg.libX11
+    xorg.libXext
+    qt6.qtbase
+    qt6.qtdeclarative
+    qt6.qtwayland
+    vulkan-headers
+    vulkan-loader
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    qt6.wrapQtAppsHook
+  ];
+
+  postPatch = ''
+    substituteInPlace src/common/path_util.cpp \
+      --replace-fail 'const auto user_dir = std::filesystem::current_path() / PORTABLE_DIR;' \
+      'const auto user_dir = std::filesystem::path(getenv("HOME")) / ".config" / "shadPS4";'
+  '';
+
+  configurePhase = ''
+    cmake -B build -DCMAKE_BUILD_TYPE=DEBUG -DENABLE_QT_GUI=ON
+  '';
+
+  buildPhase = ''
+    cmake --build build --config Release --parallel
+  '';
+
```

Comment:
1. The `cmake` setup hook already handles calling this.
2. Passing `--parallel` as-is just completely disregards `NIX_BUILD_CORES`, we really don't want that.

```suggestion
```